### ISSUE TITLE
web3.eth.sendTransaction not receive after upgrading MetaMask from 4.11.x to 4.12.0?

### ISSUE THREAD

**Issue by hoangdv1995** [ethereum/web3.js#1978]
```
web3.eth.sendTransaction({from, to, value})
.then(function(receipt){
    console.log(receipt);   // not working
});
// OR
web3.eth.sendTransaction({from, to, value})
.on('receipt', function(receipt) {
    console.log(receipt);   // not working
})
```

**Comment by vibern0** [comment]
Interesting that, after seeing this issue, I checked my metamask version, and realized it's 4.9, and the 4.12 is only available for chrome. In the next day, someone asked for help on the truffle gitter with an error like `TypeError: Cannot read property 'connection' of undefined`. And suddenly I thought about this issue. A few answers late I said `afaik there are people having some kind of trouble with metamask 4.12` which in response I got `downgrading to metamask 4.11.1 worked!`.
So, looks like there is a problem, but it is with metamask, not with web3.js
